test/fit_tests/common/fit_common.py

```diff
@@ -666,29 +671,28 @@ def apply_obm_settings_new():
         count += 1
 
     # run each OBM credential workflow on each node in parallel until success
-    nodelist = node_select()
     nodestatus = {} # dictionary with node IDs and status of each node
-    for node in nodelist:
-        nodestatus[node]= {"status": "pending", "instanceId": "", "sku": get_node_sku(node), "retry": 0}
-    for dummy in range(0, 60):
+    for dummy in range(0, 30):
```

Comment:
where does `30` (or the previous `60`) come from in terms of length? Would this be better either being passed in a argument to this function or deterministic based on the number of nodes discovered in the system?